task_templates/2_estimators/12_python_pretrained/README.md

```diff
@@ -0,0 +1,6 @@
+# Using a pretrained model in Python
+
+This template shows how to utilize a pretrained model in a custom task.  The pretrained model
+in this case is a simple linear regression model trained on the [Juniors 3 year stats dataset](tests/testdata/juniors_3_year_stats_regression.csv).  It 
```

Comment:
This dataset is only available in DR repo and so any non-DR user of DRUM won't have access to this CSV link

Reply:
That is part of this (drum) repo, which is not restricted.

Reply:
Oh gotcha thanks for the clarification